extensions/hackmd/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hackmd",
+  "title": "HackMD",
+  "description": "HackMD Raycast extension",
+  "icon": "command-icon.png",
+  "author": "Yukai",
+  "categories": [
+    "Productivity",
+    "Web",
+    "Applications",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "browse-notes",
+      "title": "Browse Notes",
+      "subtitle": "HackMD",
+      "description": "Browser HackMD Notes",
+      "mode": "view"
+    },
+    {
+      "name": "view-release-notes",
+      "title": "View Release Notes",
+      "subtitle": "HackMD",
+      "description": "View HackMD Release Notes",
+      "mode": "view"
+    },
+    {
+      "name": "create-note",
+      "title": "Create Note",
+      "subtitle": "HackMD",
+      "description": "Create a HackMD Note",
+      "mode": "view"
+    },
+    {
+      "name": "view-history",
+      "title": "View History",
+      "subtitle": "HackMD",
+      "description": "View HackMD History",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "type": "password",
+      "title": "HackMD API Token",
+      "description": "The API token you created on HackMD",
+      "name": "api_token",
+      "required": true
+    },
+    {
+      "type": "textfield",
+      "title": "HackMD API Base URL",
+      "description": "The base URL of HackMD API",
+      "name": "api_base_url",
+      "default": "https://api.hackmd.io/v1",
```

Comment:
isn't this always the same or it varies for every HackMD user/account?

Reply:
HackMD has an enterprise edition for self-hosting instances, so I leave an option here.